src/lib/integrations/daoStack/arc/schemes.ts

```diff
@@ -45,6 +45,21 @@ export const schemes: Scheme[] = [
       return [votingMachineParametersKey, votingMachineAddress]
     },
   },
+  {
+    typeName: "GenericScheme",
+    displayName: "DAO can call external contracts",
+    description:
+      "Proposals that, when passed, can call external functions in a generic manner.",
+    toggleDefault: true,
+    permissions: "0x00000000" /* no permissions */,
+    params: [],
+    getCallableParamsArray: function(
```

Comment:
I'm thinking about creating a votingCallbackFunction that we use for all schemes that use the voting machine, thoughts?

Reply:
I am not sure I follow, could you elaborate. How would it work?

As it is set up now, It is possible to select a voting machine individually for each Scheme (because of https://github.com/dOrgTech/DAOcreator/issues/75#issuecomment-478911473).

Reply:
It just seemed that regardless of which voting machine you chose, the callbacks would be the same.

Reply:
Aha, I see. But some schemes take in different parameters for the voting machines so I dont think we can do that